src/App.tsx

```diff
@@ -34,62 +35,123 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((a, b) => {
+    switch (sortType) {
+      case SortType.LENGTH:
+        return a.length - b.length;
```

Comment:
It would be nice to use the `switch`  first and then sort all the cases
```suggestion
  switch (sortType) {
    case SortType.LENGTH:
      visibleGoods.sort(a, b) => (
        return a.length - b.length;
```